expo/expo
# Polyfill console logging in bare workflow

# Why

If the app is being run outside of the Expo Go app and not using `expo-dev-menu`, then we can attempt to polyfill the `logUrl` to enable console logging in the CLI.



# How

Check if the manifest is defined, if so, then get the dev server URL and append the `logs` path component. This will ensure logs get sent to the console when dev menu isn't installed. 



# Test Plan

- Copy into a bare project that doesn't have `expo-dev-menu` installed.
- Run the project with `expo start --dev-client` or use `expo run:ios`, `expo run:android` (RN CLI logging still works).
- Run the project with `npx react-native start` to ensure that logs aren't being printed twice.
